### PR TITLE
Update docs to include CUDA shared memory

### PR DESCRIPTION
**Pull request by CoderHam**

### CODE REVIEW

File: docs/client.rst [hunk 1]
```diff
@@ -310,23 +310,23 @@ and then::
...
-To run the the Python version of the simple example, first build or
+To run the Python version of the simple example, first build or
 download it as described in :ref:`section-getting-the-client-examples`
 and install the tensorrtserver whl, then::
 
   $ python simple_client.py
 
-Shared Memory
+System Shared Memory
 ^^^^^^^^^^^^^
```
Comment (deadeyegoodwin) [inline docs/client.rst]: Extend the "^" to length of heading

File: docs/client.rst [hunk 2]
```diff
@@ -342,13 +342,52 @@ build or download it as described in
...
+memory. To run the Python version of the simple system shared memory example,
 first build or download it as described in
 :ref:`section-getting-the-client-examples` and install the
-tensorrtserver whl, then::
+tensorrtserver whl and then::
 
   $ python simple_shm_client.py
 
+CUDA Shared Memory
+^^^^^^^^^^^^^
```
Comment (deadeyegoodwin) [inline docs/client.rst]: Extend

File: docs/client.rst (hunk 1 above)
Comment (CoderHam) [reply docs/client.rst]: Done

File: docs/client.rst (hunk 2 above)
Comment (CoderHam) [reply docs/client.rst]: Done

**Pull request merged**